woocommerce/woocommerce-android
# Remove order creation announcement

### Description
This PR removes the announcement banner of the  Order Creation feature as It's have been two years since we released the feature.

### Testing instructions
1. Open the orders lists screen
2. Check that the Order creation announcement is missing

### Images/gif

| Before  | After |
| ------------- | ------------- |
| <img width="300" src="https://github.com/woocommerce/woocommerce-android/assets/18119390/581c3571-892f-4688-8840-f4ad273d083a" /> | <img width="300" src="https://github.com/woocommerce/woocommerce-android/assets/18119390/72c47d2f-7732-4669-938b-ef6b154c4d90" />  |
